app/models/incident.rb

```diff
@@ -120,44 +134,71 @@ def reviewed?
         state: 'MA',
-        # zip: driver.incident_report.zip,
+        zip: report.zip,
         longitude: longitude,
         latitude: latitude,
-        'IncidentDesc' => root_cause_analysis,
-        'VehicleRouteNum' => report.block,
-        'VehicleNum' => 777, # will be pulled from claims table
+        # AccidentCode, AccidentCodeGroup
+        'Company' => driver.division.claims_id,
+        'IncidentDesc' => supervisor_incident_report.try(:description),
+        'EmployeeID' => driver.badge_number,
+        'Driver' => 999, # TODO: pull from claims table
+        'DriverDesc' => report.description,
+        'VehicleRouteNum' => report.route,
+        # VehicleDestination
+        'VehicleNum' => 777, # TODO: pull from claims table
+        # VehicleAppraisalAmnt
+        'VehicleDamageArea' => report.damage_to_other_vehicle_point_of_impact,
+        # Comments, ReportGiver
         'PointOfContact' => report.damage_to_bus_point_of_impact,
+        # TotalSettlement
         'Status' => 'ir',
+        # CloseDate, ReopenDate, DownDate, ac_type
         reason1: reason_code.identifier,
         reason2: second_reason_code.try(:first, 3),
-        'Company' => driver.division.claims_id,
-        'DateEntered' => Date.today.strftime('%Y-%m-%d'),
-        'IncidentDate' => report.occurred_at,
-        'EmployeeID' => driver.badge_number,
-        'Driver' => 999, # will be pulled from claims table
-        'DriverDesc' => report.description,
       },
       drivers_report: {
-        'Speed' => report.speed,
+        'FileID' => claims_id,
+      # 'PolicePresent' => report.police_on_scene?,
+        # OfficerName
+      # 'BadgeNum' => report.police_badge_number,
+        # ArrivalTime
+        'Citation' => report.summons_or_warning_issued?,
+      # 'CitationWho' => report.summons_or_warning_info,
         'Weather' => report.weather_conditions,
+        'SurrCond' => report.road_conditions,
         'Lighting' => report.light_conditions,
+        # LossLocation
+        'Speed' => report.speed,
+        'MotionBus' => report.motion_of_bus,
+      # 'Direction' => report.direction,
+        # ChairOnLift, LiftDeployed, PassengersPresent, SeatBelts
         'PointOfContact' => report.damage_to_bus_point_of_impact,
-        'FileID' => 888, # will be pulled from corresponding claims record
+      # 'CurbDist' => report.bus_distance_from_curb,
+        # VehicleDistance, ReviewFlg
         'TotalPass' => report.passengers_onboard,
-        # 'Ambulance' => report.injured_passengers.any?(&:transported_to_hospital?)
+        # PVTAatFault, Wheelchair, YellowLine, NotReported
+        'Ambulance' => report.injured_passengers.any?(&:transported_to_hospital?),
+        'OVTow' => report.other_vehicle_towed_from_scene?,
+        'PVTATow' => report.towed_from_scene?,
+        # Fatality, assistRequest, PD, Note, externalAppraisal
       },
-    }.fetch(table)
-
-    arel_insert_statement(table, fields)
+    }
+    
+    fields.fetch(table)
```

Comment:
Why do you prefer this over what was there previously? Just to cut down on complexity and readability? Or?

Reply:
I think a long hash with a method chained at the end can be a little sneaky, yes. Definitely expressing more than one idea at once.